VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -367,4 +375,73 @@ public void orientationChangeTest() {
                 activity.getResources().getConfiguration().orientation);
         assertOrientationSetup(activity);
     }
+
+    /**
+     * Testing creation of various fragments in the {@link CourseUnitNavigationActivity}'s
+     * ViewPager, by supplying its {@link CourseUnitPagerAdapter} with all possible
+     * {@link CourseComponent} models.
+     */
+    @Test
+    public void testUnitFragmentCreation() {
```

Comment:
This will probably need to go in the Espresso test suites instead of the Robolectric ones then, because Robolectric [doesn't have][1] a parameterized test suites at the moment. While we're mocking everything, the `CourseUnitPagerAdapter` initialization references classes reference Android-specific classes, e.g. `Bundle`.

[1]: https://github.com/robolectric/robolectric/issues/1934